internal/cli/atlas/atlas.go

```diff
@@ -102,3 +110,14 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+func newDeprecatedMessage() string {
```

Comment:
[nit] `new` is for constructors 
```suggestion
func deprecatedMessage() string {
```